org.civicrm.reqangular/src/common/templates/timepicker-select.html

```diff
@@ -0,0 +1,2 @@
+<option ng-if="selector.defaultOption" value="">{{ selector.defaultOption }}</option>
```

Comment:
Can we call it placeholder? Default Option makes me think that is the time that will be automatically selected.